extensions/binding/org.eclipse.smarthome.binding.sonos/ESH-INF/config/config.xml

```diff
@@ -9,11 +9,11 @@
 			<description>The UDN identifies the Zone Player.</description>
 			<required>true</required>
 		</parameter>
-		<parameter name="notificationTimeout" type="integer">
+		<parameter name="notificationTimeout" type="integer" unit="s">
 			<label>Notification Timeout</label>
-			<description>Specifies the amount of time for which the notification sound will be played</description>
-			<default>40</default>
-			<required>true</required>
+			<description>Specifies the amount of time in seconds for which the notification sound will be played</description>
+			<unitLabel>second</unitLabel>
```

Comment:
According to the code the tag `unitLabel` is ignored if the attribute `unit` is set (see [https://github.com/eclipse/smarthome/blob/master/bundles/config/org.eclipse.smarthome.config.xml/src/main/java/org/eclipse/smarthome/config/xml/ConfigDescriptionParameterConverter.java#L120](https://github.com/eclipse/smarthome/blob/master/bundles/config/org.eclipse.smarthome.config.xml/src/main/java/org/eclipse/smarthome/config/xml/ConfigDescriptionParameterConverter.java#L120). Isn't it?